ui/Rhino/AGS/dev/AGS_force_scale_cmd.py

```diff
@@ -20,10 +20,14 @@ def RunCommand(is_interactive):
     scene = sc.sticky['AGS']['scene']
     force = scene.find_by_name('Force')[0]
 
-    scale_factor = rs.GetReal("Scale factor", force.artist.scale)
-    force.artist.scale = scale_factor
+    vertex = force.select_vertex(message="Pick base node.")
+    if vertex:
+        force.artist.anchor_point = force.artist.vertex_xyz[vertex]
+        force.artist.anchor_vertex = vertex
+
+        scale_factor = rs.GetReal("Scale factor", force.artist.scale)
```

Comment:
Can we use `compas_rhino.rs.GetReal` here? Such as we don't need to import `rs`? This seems to be a preference for this package.

Reply:
sorry